### PR TITLE
Add package readme to System.Numerics.Tensors

### PR DESCRIPTION
**Pull request by ericstj**

### CODE REVIEW

**Review by jeffhandley — APPROVED** [review]
LGTM. @michaelgsharp / @tannergooding -- are we using the words "vector" and "tensor" correctly in each occurrence?

**Review by michaelgsharp — APPROVED** [review]

File: src/libraries/System.Numerics.Tensors/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+## About
+
+Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use vector operations supported by the CPU where available.
+
+## Key Features
+
+* Numerical operations on Vectors represented as `ReadOnlySpan<float>`
```
Comment (ericstj) [inline src/libraries/System.Numerics.Tensors/src/PACKAGE.md:7]:
```suggestion
* Numerical operations on tensors represented as `ReadOnlySpan<float>`
```

File: src/libraries/System.Numerics.Tensors/src/PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+## About
+
+Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use vector operations supported by the CPU where available.
```
Comment (stephentoub) [inline src/libraries/System.Numerics.Tensors/src/PACKAGE.md:3]:
```suggestion
Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use SIMD operations supported by the CPU where available.
```
?

File: src/libraries/System.Numerics.Tensors/src/PACKAGE.md (hunk 1 above)
Comment (stephentoub) [inline src/libraries/System.Numerics.Tensors/src/PACKAGE.md:7]:
```suggestion
* Numerical operations on tensors represented as `ReadOnlySpan<float>`
```

File: src/libraries/System.Numerics.Tensors/src/PACKAGE.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Key Features
+
+* Numerical operations on Vectors represented as `ReadOnlySpan<float>`
+* Element-wise arithmetic: Add, Subtract, Multiply, Divide, Exp, Log, Cosh, Tanh, etc.
+* Vector arithmetic: CosineSimilarity, Distance, Dot, Normalize, Softmax, Sigmoid, etc.
```
Comment (stephentoub) [inline src/libraries/System.Numerics.Tensors/src/PACKAGE.md:9]:
```suggestion
* Tensor arithmetic: CosineSimilarity, Distance, Dot, Normalize, Softmax, Sigmoid, etc.
```

**Review by stephentoub — APPROVED** [review]
A few suggestions, but Tanner should really review.

File: src/libraries/System.Numerics.Tensors/src/PACKAGE.md (hunk 2 above)
Comment (ericstj) [reply src/libraries/System.Numerics.Tensors/src/PACKAGE.md:3]:
```suggestion
Provides methods for performing mathematical operations over _tensors_ represented as spans.  These methods are accelerated to use SIMD (Single instruction, multiple data) operations supported by the CPU where available.
```

**Comment by ericstj** [comment]
Would like to get this in before today's cutoff.  @tannergooding I'll hold for your feedback for 30min, then I'll merge and port this to 8.0.

**Pull request merged**